docs/database/README.md

```diff
@@ -109,6 +111,26 @@ db.query("select name from contacts where age = ?", 33) do |rs|
 end
 ```
 
+Query parameters are effected under the covers with prepared statements (sometimes cached),
+or insertion on the client side, depending on the driver, but will always avoid SQL Injection.
+
+If you want to manually use prepared statements, you can with the `build` method:
+
+```crystal
+# MySQL
+prepared_statement = db.build("select * from contacts where id=?")
+# Postgres
+prepared_statement = db.build("select * from contacts where id=$1")
```

Comment:
I'd suggest to use different names for the databases to avoid any confusion.
```suggestion
prepared_statement = mysql_db.build("select * from contacts where id=?")
# Postgres
prepared_statement = postgres_db.build("select * from contacts where id=$1")
```

Reply:
The example doesn't work anymore when you only rename the statement variables. The following lines should also use the new names. Sorry, I didn't include that into my suggestion.
It was also meant to start a discussion about improvements.
I think the best way would actually be to show only a single example using one driver-specific syntax (and mentioning alternatives in a comment).

```crystal
prepared_statement = mysql_db.build("select * from contacts where id=?") # Use "... where id=$1" for PostgreSQL
# Use prepared statement:
prepared_statement.query(3) do |rs|
  # ... use rs
end
prepared_statement.query(4) do |rs|
  # ... use rs
end
prepared_statement.close
```

Reply:
Yes, I think it doesn't work as it is now. @rdp can you change that? Otherwise, we'll do it. With that, this is ready. Thanks and sorry for the confusion.

Reply:
all righty pushed an update.  I didn't rename it to mysql_db since it doesn't do that elsewhere in the file so felt like it would make sense as part of a bigger change...